verifact/forum/fixtures/questions.json

```diff
@@ -1,24 +1,26 @@
 [
     {
-        "model":"forum.question",
-        "pk":1,
-        "fields":{
+        "model": "forum.question",
```

Comment:
hey, I don't think we use the files in /forum/fixtures anymore to generate sample data - instead, as of #12 we've moved to `pytest` and factories made from `factory-boy` for testing. (you can look at `verifact/tests/factories.py`)

Reply:
Actually, I included a section on how to use fixtures to pre-load data into the database, which is probably what @jtan381 is trying to do. That's my mistake, it's been so long since I had used them and thought they were only used in tests. It makes sense to keep them after all, @jtan381 can you resolve the conflicts first? Thanks.

Reply:
oh, I see, makes sense. should we restore all the previous fixtures in this PR then? and write some Vote fixtures as well

Reply:
Probably a good idea, yeah.